Sources/TabBar/View/TabBar.swift

```diff
@@ -96,7 +97,7 @@ public struct TabBar<TabItem: Tabbable, Content: View>: View {
                     }
                 }
                 .edgesIgnoringSafeArea(.bottom)
-                .visibility(self.visibility.wrappedValue)
+                .visibility(self.$visibility.wrappedValue)
```

Comment:
You can use just `self.visibility` here

Reply:
I thought so too. But then I get the following error:

<img width="1013" alt="grafik" src="https://user-images.githubusercontent.com/36701166/156532880-9a1a9554-88c2-459a-81ae-7d74de5f1332.png">

Reply:
I mean, you do not need to use `.wrappedValue` property, because you use `@Binding` property wrapper

Reply:
Ahh, I see. 

I'm not at home right now. I'll put it in later.